bindings/python/src/OpenSpaceToolkitPhysicsPy/Time/Interval.cpp

```diff
@@ -191,7 +191,7 @@ inline void OpenSpaceToolkitPhysicsPy_Time_Interval(pybind11::module& aModule)
                 Get the intersection with another interval.
 
                 Args:
-                    interval (Interval): Another interval.
+                    (Interval): Another interval.
```

Comment:
I think you misunderstood the format for the Args field, you have to indeed provide the name of the argument, the type and then the description. This is correct IMO.

Reply:
discusssed that this is because there is no kwarg here